server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -272,17 +326,21 @@ function validateClientInformation($token, $firstName, $lastName, $emailAddress,
 function getClientInformationFromToken($token) {
 	GLOBAL $DB_CONN;
 
+	// LEFT JOIN consent so that if consent isn't found, it still returns the rest of the info
 	$query = 'SELECT Client.firstName, Client.lastName, Client.emailAddress, Client.phoneNumber, Client.bestTimeToCall, 
 			DATE_FORMAT(AppointmentTime.scheduledTime, "%W, %M %D at %l:%i %p") AS appointmentTimeStr, 
 			DATE_FORMAT(DATE_SUB(AppointmentTime.scheduledTime, INTERVAL 7 DAY), "%W, %M %D") AS uploadDeadlineStr,
 			AppointmentTime.scheduledTime, Appointment.appointmentId, Appointment.language,
-			AppointmentTime.siteId, AppointmentType.lookupName AS appointmentType, Site.title
+			AppointmentTime.siteId, AppointmentType.lookupName AS appointmentType, Site.title,
+			(VirtualAppointmentConsent.virtualConsent = 1 AND NOT (VirtualAppointmentConsent.signature IS NULL OR /*column is set to NOT NULL. Left it to be sure*/
+			VirtualAppointmentConsent.signature = \'\')) as consented
```

Comment:
Hm, I think I would sooner pull out the virtual appointment information and have a PHP function to check `consented`. This way, you could use that same function above in `storeConsent` to return `response['consented']`. Then we could also do the `trim` and whatnot